src/metaswitch/crest/api/homestead/provisioning/handlers/public.py

```diff
@@ -76,10 +76,15 @@ def get(self):
 
         # Query all subscribers, chunk-by-chunk, and stream it back to the
         # client
-        first = True
+        first_result = True
+        first_chunk = True
```

Comment:
I guess not but it's weird to sleep as the first thing in the loop. I'll move the sleep to the end and remove first_chunk.